src/main/java/io/github/jhipster/registry/config/OAuth2SecurityConfiguration.java

```diff
@@ -78,16 +69,19 @@ public void configure(HttpSecurity http) throws Exception {
             .headers()
             .frameOptions()
             .disable()
+        // TODO: Should be 'STATELESS' if (angular) client is oidc/oauth2 aware
+        // Currently, we 'store' the oidc/oauth2 stuff with http session
```

Comment:
We've discussed this many times. I'm against putting any oauth2 logic in clients. It reduces security because JWTs are stored in local storage.

Reply:
I'm in line with you to remove the 'TODO' line, as this discussion has nothing to do with the problem at hand.